sklearn/ensemble/tests/test_bagging.py

```diff
@@ -966,3 +967,61 @@ def test_n_features_deprecation(Estimator):
 
     with pytest.warns(FutureWarning, match="`n_features_` was deprecated"):
         est.n_features_
+
+
+@pytest.mark.parametrize(
+    "BaggingEstimator, load_data",
+    [
+        (
+            partial(BaggingClassifier, base_estimator=DecisionTreeClassifier()),
```

Comment:
just a nitpick, but you could parametrize over "bagging_estimator, base_estimator" and then
```py
    bagged_trees = BaggingEstimator(
        base_estimator=base_estimator,
        n_estimators=n_estimators,
        max_features=1.0,
        bootstrap_features=bootstrap_features,
        random_state=rng,
    ).fit(X_train, y_train)
```